Mvc.sln

```diff
@@ -803,7 +806,6 @@ Global
 		{42C5D417-4060-48F4-BB28-E9E179007779}.Release|Mixed Platforms.Build.0 = Release|Any CPU
 		{42C5D417-4060-48F4-BB28-E9E179007779}.Release|x86.ActiveCfg = Release|Any CPU
 		{42C5D417-4060-48F4-BB28-E9E179007779}.Release|x86.Build.0 = Release|Any CPU
-<<<<<<< HEAD
```

Comment:
Yupp I need to fix it, but I thought I'll do it when I am ready to check in once and for all.